dali/python/nvidia/dali/backend.py

```diff
@@ -38,5 +38,13 @@
                           "reaches the end of life on January 1st, 2020. To stay up to date with "
                           "DALI, please upgrade to Python 3.5 or later.", Warning, stacklevel=2)
 
+    # TODO: remove in 0.20
+    def old_at(self, id):
```

Comment:
> Can we make this on the backend_impl level instead of through setattr?

Not easily, we would have to factorize it out (can't do the `setattr` trick directly in pybind, as least from what I read)
> The deprecation warning uses TensorListGPU and TensorGPUList in the same sentence.
Thanks, will change that!

Reply:
> Can we make this on the backend_impl level instead of through setattr?

Not easily, we would have to factorize it out (can't do the `setattr` trick directly in pybind, as least from what I read)
> The deprecation warning uses TensorListGPU and TensorGPUList in the same sentence.

Thanks, will change that!